pkg/api/v1alpha1/cluster_webhook_test.go

```diff
@@ -1242,6 +1242,7 @@ func TestClusterCreateManagementCluster(t *testing.T) {
 
 func TestClusterCreateCloudStackMultipleWorkerNodeGroupsValidation(t *testing.T) {
 	os.Setenv(features.CloudStackProviderEnvVar, "true")
+	os.Setenv("FULL_LIFECYCLE_API", "true")
```

Comment:
I would suggest setting the env var this way:

https://github.com/aws/eks-anywhere/blob/main/pkg/clustermanager/cluster_manager_test.go#L932

Reply:
Fixed, thank you. I've been struggling with this for a while now